build.xml

```diff
@@ -414,8 +414,8 @@
         debug="${javac.debug}"
         deprecation="${javac.deprecation}"
         includeantruntime="false"
-        target="1.6"
-        source="1.6">
+        target="11"
+        source="11">
```

Comment:
I propose that when we "merge" the `tls_support` branch, we bump the main branch's minor version so that we don't interfere with projects that might be depending on the old branch.